pxtlib/emitter/cloud.ts

```diff
@@ -43,17 +73,7 @@ namespace pxt.Cloud {
             options.headers["x-td-access-token"] = accessToken
```

Comment:
is this token dead?

Reply:
not sure; I guess we might want a token if we ever have a login